README.md

```diff
@@ -1,5 +1,7 @@
 # terraform-cloudwatch
 
+N.B. Releases >= 2.0.0 been upgraded to use Terraform 0.12.?
```

Comment:
Why the exclamation mark?

Reply:
Changed to an `n` to make it less ambiguous.